test/testData.js

```diff
@@ -73,6 +73,26 @@ exports.initialPlayerState = {
     maxMana: 10,
     name: 'Walker',
   },
+  playerId3: {
```

Comment:
Use `playerId1` or `playerId2` for this test.

We should try to write the least amount of test data possible. That way, when we want to update the shape of our state there will be less code we will have to change.